app/main.py

```diff
@@ -0,0 +1,13 @@
+from fastapi import FastAPI
+from fastapi.staticfiles import StaticFiles
+
+from app.routes import books
+from app.routes import users
```

Comment:
Можно писать через запятую
```suggestion
from app.routes import books, users
```